help/access/file-shares.md

```diff
@@ -1,302 +1,237 @@
+The **File Manager** is an interface that allows you to manage connections, browse and preview file content, and perform standard file and folder actions such as opening, downloading, deleting, and renaming. To access an object's context actions, right-click it or left-click and expand the **Actions** pane in the **Context Panel** on the left. By clicking a file or folder in the **File Manager**, you can open its preview. Double-clicking a file opens it in Datagrok, and double-clicking a folder expands its content.
 
-4. Optionally, enter a description in the text field provided. You may also notify the users you share with. If you
-   don’t want to send a notification, uncheck the **Send notification** checkbox.
+:::note
 
-   > Note: To send an email notification, enter the user's email in the identity/email field. The email notification
-   > contains a link to the shared item and entered description. If you enter a user or group name, they will be
-   > notified via the Datagrok interface.
+If you don't see a certain action, it may be due to insufficient permissions. For files and folders shared with you, contact the credentials owner. If you are a credentials owner, contact the data source owner.
 
-5. Click **OK** to share. Once shared, the shared item appears in the recipient's **Folder Tree**.
+:::
 
-   ![Share a folder](share-the-folder.gif)
+In addition to the hierarchical browsing, the **File Manager** offers an array of advanced preview and data augmentation capabilities using **Directory**, **Preview**, and **Context Panel**.
 
-   > Note: When you share a file or a folder, Datagrok automatically indexes folders <!--link out to indexing when ready-->
-   > and working--> and extracts basic metadata (like the date created or its size). File indexing is optional (to index
-   > files, toggle **Index Files** in the **Share...** dialog).
+The **Directory** section shows the contents of your current folder with three viewing modes: icons, cards, and grid. You can click an object to view its content in the **Preview**, or right-click it to access available context actions. You can also use the search bar located above the **Directory** to search for files and folders within your current directory. The search bar allows you to search for items by name, file extension, or metadata.
 
-## Manage file shares
+The **Preview** is a context-sensitive view that adapts to the the selected object. For folders, the **Preview** generates a [treemap](../visualize/viewers/tree-map.md) that highlights the largest items. For files, the functionality varies based on the file's format and data properties. It includes custom viewers for [supported formats](supported-formats.md), such as interactive spreadsheets for displaying tabular data, cell and image renderers, and chemical and biological structure viewers. You can also view the content of ZIP files and edit Markdown, TXT, and HTML files.
```

Comment:
The **Preview** is a context-sensitive view that adapts to the the selected object.
=> 
It's sort of implied by the name "Preview". Drop it altogether

Reply:
For folders, the Preview generates a [treemap](http://localhost:3001/help/visualize/viewers/tree-map) that highlights the largest items
=>
focus on files, this is the most important thing. Mention the folders later.

Folder preview is a size-coded tree map of its content, so you see what takes the most disk space.

Reply:
"For files, the functionality varies based on the file's format and data properties."
=> (something along these lines)
The Datagrok ecosystem supports 50+ file formats, and previews file content in the best way possible: an interactive spreadsheet for tabular data, a 3D molecule viewer for chemical structures, a rendering of a PDF document, etc

Reply:
I don't like your version. I think you have a familiarity bias here and not enough info for users.